swagger.yml

```
@@ -0,0 +1,1098 @@
+        $ref: '#/definitions/prozent'
+      sollzinssatz:
+        $ref: '#/definitions/prozent'
+      gesamtbetrag:
+        description: Auszahlungsbetrag + Versicherungsprämie + Zinsen
+        $ref: '#/definitions/euro'
+      nettokreditbetrag:
+        description: Auszahlungsbetrag + Versicherungsprämie
+        $ref: '#/definitions/euro'
+      auszahlungsbetrag:
+        description: Kreditbetrag
+        $ref: '#/definitions/euro'
+      laufzeitInMonaten:
+        type: integer
+      rateProMonat:
+        $ref: '#/definitions/euro'
+      letzteRate:
+        $ref: '#/definitions/euro'
+      provision:
+        $ref: '#/definitions/euro'
+      basisbetragFuerDieProvisionsermittlung:
+        $ref: '#/definitions/euro'
+      vorlaufzinsenProTag:
+        $ref: '#/definitions/euro'
+    required:
+      - effektivzinssatz
+      - sollzinssatz
+      - gesamtbetrag
+      - nettokreditbetrag
+      - auszahlungsbetrag
+      - laufzeitInMonaten
+      - rateProMonat
+      - letzteRate
+      - provision
+      - basisbetragFuerDieProvisionsermittlung
+      - vorlaufzinsenProTag
+
+  versicherungskonditionen:
+    type: object
+    properties:
+      basisFuerDieProvisionsermittlung:
+        type: string
+        enum:
+          - VERSICHERUNGS_PRAEMIE_NETTO
+          - VERSICHERUNGS_PRAEMIE_BRUTTO
+      versicherungsprovision:
+        $ref: '#/definitions/euro'
+      versicherungspraemieBrutto:
+        description: Versicherungsprämie inklusive Steuer für Arbeitslosigkeitsversicherung
+        $ref: '#/definitions/euro'
+      versicherungspraemieNetto:
+        description: Versicherungsprämie exklusive Steuer für Arbeitslosigkeitsversicherung
+        $ref: '#/definitions/euro'
+      versicherungsrisiken:
+        type: array
+        items:
+          $ref: '#/definitions/versicherungsrisiken'
+      versicherer:
+        type: string
+    required:
+      - basisFuerDieProvisionsermittlung
+      - versicherungsprovision
+      - versicherungspraemieBrutto
+      - versicherungspraemieNetto
+      - versicherungsrisiken
+      - versicherer
+
+  meldung:
+    type: object
+    properties:
+      kategorie:
+        type: string
+        enum:
+          - MACHBARKEIT
+          - VOLLSTAENDIGKEIT
+          - HINWEIS
+          - ANPASSUNG
+      text:
+        type: string
+      code:
+        type: string
+    required:
+      - kategorie
+      - text
+      - code
+
+
+  bonitaetscheckPosition:
+    type: object
+    properties:
+      positionen:
+        type: array
+        items:
+          type: object
+          properties:
+            label:
+              type: string
+            wert:
+              $ref: '#/definitions/euro'
+          required:
+            - label
+            - wert
+    required:
+      - positionen
+
+  dokument:
+    type: object
+    properties:
+      bezeichnung:
+        type: string
+      dateiname:
+        type: string
+      base64pdf:
+        type: string
```

Format für Base64 Kodierung angeben

```suggestion
        type: string
        format: byte
```